DNSSD/.gitignore

```diff
@@ -0,0 +1,153 @@
+# Compiled #
```

Comment:
there's no need for this file, there is the top-level one

Reply:
@aleks-f  I remove it